File: Assignment_40/program4.c
//////////////////////////////////////////////////////////////////////////
//
// Name :          program4.c
// Discription :   Get input and Display
// Output :        Charecter (A B C D)
// Author :        Shivam Rajendra Kale
// Date :          11/07/2025
//
//////////////////////////////////////////////////////////////////////////

/* 
  Start
      Accept input from user
          Display  A B C D 
  Stop    
*/

#include<stdio.h>

void Display(char iSize)
{
    static int count = 1;  
    static char ch = 'A';
    

    if(count <= iSize)
    {   
        printf("%c\t",ch);

        count++;
        ch++;
    
        Display(iSize);
    }
    else 
    {
        return;
    }
}

int main()
{
    int iValue = 4;

    Display(iValue);
   
    return 0;
}

File: Assignment_54/program4.java
//////////////////////////////////////////////////////////////////////////
//
// Name :          program4.java
// Discription :   use to Display class of each student
// Input :         Integer
// Output :        Integer
// Author :        Shivam Rajendra Kale
// Date :          18/10/2025
//
//////////////////////////////////////////////////////////////////////////

import java.util.*;

class MyArray
{
    public void RevarseArray(float Arr[])
    {
        int i = 0;

        // code
    }
}

class program4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int i = 0;

        System.out.println("Enter isze of first Array : ");
        int iSize = sobj.nextInt();

        float Marks[] = new float [iSize];

        System.out.println("Enter elements : ");
        for(i = 0; i < iSize; i++)
        {
            Marks[i] = sobj.nextFloat();
        }

        MyArray mobj = new MyArray();

        mobj.RevarseArray(Marks);

        sobj.close();
    }
}